.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+    name: cargo check/clippy/fmt/test
+    runs-on: ubuntu-latest
+
+    env:
+      CARGO_TERM_COLOR: always
+      RUST_BACKTRACE: 1
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - run: rustup toolchain install stable --profile minimal
```

Comment:
Aren't all tools already installed?

https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md

Reply:
Thanks for looking into that, it looks like they are. The last time I looked into this, rust either wasn't installed or was a couple versions behind.
I'll remove that step.